packages/teams-js/package.json

```diff
@@ -24,8 +24,8 @@
   },
   "devDependencies": {
     "@types/debug": "4.1.7",
-    "@fluidframework/azure-client": "~1.0.1",
-    "fluid-framework": "~1.2.2"
+    "@fluidframework/azure-client": "~1.0.2",
```

Comment:
I believe these two need to move to being `dependencies` instead `devDependencies` so that they are picked up during install calls by anyone consuming the package

Reply:
I was able to remove them all together. We don't want them under `dependencies`  as they would increase the size of the teams-js SDK by a factor of 10.